movie/src/components/Card/index.jsx

```diff
@@ -2,29 +2,71 @@ import "./Card.css";
 import Grid from "@mui/material/Grid";
 import { Item } from "semantic-ui-react";
 import DetailIcon from "../DetailIcon";
+import months from "../../data/Month.json";
+import { useState } from "react";
 const IMAGE_URL =
   "https://images.unsplash.com/photo-1503023345310-bd7c1de61c7d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwcm9maWxlLXBhZ2V8N3x8fGVufDB8fHx8&w=1000&q=80";
 
 const Card = ({ movies }) => {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+  const newMovies = [...movies];
+  const [idCardClicked, setIdCardClicked] = useState(0);
```

Comment:
My guesses is that `idCardClicked` is use to store an `id` right? 

if that that the case the name should be `cardId` || `currentCardId`. 

`idCardClicked` mean its has the value of a Boolean

Reply:
Yes, I got what you mean.